EMaligner/utils.py

```diff
@@ -547,12 +770,19 @@ def get_z_values_for_stack(stack, zvals):
     return zvals[ind]
 
 
-def update_tilespecs(resolved, x, used):
+def update_tilespecs(resolved, x):
+    """update tilespecs with new solution
+    Parameters
+    ----------
+    resolved : obj
```

Comment:
For this and other specific objects or functions (numpy ndarray, renderapi transforms, EMaligner schemas, etc.), it might be an improvement to use sphinx's references intersphinx's external link capabilities.

Reply:
thanks, makes it much more useful. done.